test/e2e/storage/volume_provisioning.go

```diff
@@ -803,7 +806,10 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 		// Modifying the default storage class can be disruptive to other tests that depend on it
 		It("should be disabled by removing the default annotation [Serial] [Disruptive]", func() {
 			framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke", "vsphere", "azure")
-			scName := getDefaultStorageClassName(c)
+			scName, scErr := framework.GetDefaultStorageClassName(c)
+			if scErr != nil {
+				framework.Failf(scErr.Error())
```

Comment:
same thing, skip and remove provider check?

Reply:
Do we know that whatever default SC that will exist will support dynamic provisioning?

Reply:
it seems odd to define a default storageclass that cannot support dynamic provisioning. There is movement in the conformance area to also add some "validation" suites using the default storage class

Reply:
Given current defaults it isn't out of question to have a SC that does not support dynamic provisioning. For example - a bare metal cluster that uses only local PVCs.  The cloudprovider tests explicitly allow us to skip these tests but a default SC won't.